src/rest/Plan.ts

```diff
@@ -0,0 +1,14 @@
+import { ITrigger } from './Trigger';
+
+/**
+ * Describes the plan of execution of a search
+ */
+export interface IPlanResults {
+  processingOutput: {
+    triggers: ITrigger<any>[];
```

Comment:
Could an interface be used here instead of `any`? If there is no content, could `void` be used?

Reply:
any is used because there are many trigger types. See the Trigger.ts file for reference.